### DISCUSSION TITLE
Adding the Authelia icon to 2FAS

### DISCUSSION THREAD

**Discussion by stantios** [authelia/authelia#6131] [Ideas]
### Description

I think it would be great if the Authelia icon was added to 2FAS. 2FAS is a widely used 2FA Authentication app and I personally use it to secure all my services with 2FA and it would be great if Authelia had an icon for the 2FA code rather than having no icon.

### Use Case

The use case for this would be that there would be an icon for the 2FA service when people try to add 2FA for your service.

### Details

2FAS is an open source 2FA Authentication app that provides a way to add 2FA Authentication to all of your services. They support alot of brand icons and it would be nice if 2FAS has one as well.

### Documentation

The form is here if you want to fill it out: <https://2fas.com/your-branding/>

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
Will look into doing this soon thanks for the idea.

**Reply by Jakyrian** [reply]
Assume this hasn’t been looked into yet?
